src/vue/forms/LoginForm.spec.js

```diff
@@ -0,0 +1,188 @@
+import { TestHelper } from '@/test/test-helper'
+import LoginForm from './LoginForm'
+
+import Vuelidate from 'vuelidate'
+import Vuex from 'vuex'
+import Vue from 'vue'
+
+import { Wallet, errors } from '@tokend/js-sdk'
+import { vuexTypes } from '@/vuex'
+import { createLocalVue, shallowMount, mount } from '@vue/test-utils'
+import { globalize } from '@/vue/filters/globalize'
+
+import { MockHelper } from '@/test'
+
+// HACK: https://github.com/vuejs/vue-test-utils/issues/532, waiting for
+// Vue 2.6 so everything get fixed
+Vue.config.silent = true
+
+const localVue = createLocalVue()
+localVue.use(Vuelidate)
+localVue.use(Vuex)
+localVue.filter('globalize', globalize)
+
+describe('LoginForm component unit test', () => {
+  let mockHelper
+  beforeEach(() => {
+    sinon.restore()
+    mockHelper = new MockHelper()
+  })
+
+  describe('validation works correctly', () => {
+    let wrapper
+
+    beforeEach(() => {
+      wrapper = mount(LoginForm, { localVue })
+    })
+
+    const fields = {
+      email: {
+        valid: ['anything', 'alice@mail.com', 'qq'],
+        invalid: ['']
+      },
+      password: {
+        valid: ['anything', 'q', 'qqww'],
+        invalid: ['']
+      }
+    }
+
+    for (const [fieldName, fieldValues] of Object.entries(fields)) {
+      for (const fieldValue of fieldValues.valid) {
+        it(`considers ${fieldValue} a valid ${fieldName}`, () => {
+          expect(TestHelper.isFieldValid(
+            wrapper,
+            fieldName,
+            fieldValue
+          ))
+            .to.be.true
+        })
+      }
+      for (const fieldValue of fieldValues.invalid) {
+        it(`considers ${fieldValue} an invalid ${fieldName}`, () => {
+          expect(TestHelper.isFieldValid(
+            wrapper,
+            fieldName,
+            fieldValue
+          ))
+            .to.be.false
+        })
+      }
+    }
+
+    const fieldBindings = {
```

Comment:
You do these tests to ensure that the fields are assigned to the right models, but not sure that the current way is the best way. I mean these tests are extra tests that system not really  need because they test behavior of vendored library.

Can we test content of `@blur` tag in the rendered component instead? That could be much easier and can be automated. Even more: we could also extract such check to eslint in future.

Reply:
resolved as the effort to implement such stuff doesn' worth the result we get